tensorflow/python/ops/array_ops.py

```diff
@@ -535,19 +535,16 @@ def shape_v2(input, out_type=dtypes.int32, name=None):
   # pylint: disable=redefined-builtin
   """Returns the shape of a tensor.
 
-  This operation returns a 1-D integer tensor representing the shape of `input`.
-  This represents the minimal set of known information at definition time.
```

Comment:
I would keep this line about the minimal information

Reply:
Sure. I was going to do a second revision with the link provided by @jvishnuvardhan. Surprised this was merged already.